src/App.js

```diff
@@ -1,8 +1,190 @@
+  const currentUser = useSelector((state) => state.current_user);
+  let userPhoto = 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQyYOyr5Ec8JLXH9PnLVHZ2QKcW43XQs47vnQ&usqp=CAU';
+  if (currentUser.photo) {
+    userPhoto = currentUser.photo;
+  }
+  const adjustSize = () => {
+    const header = document.querySelector('.header');
+    const menu = document.querySelector('.menu-icon');
+    const profile = document.querySelector('.profile-icon');
+    const w = window.innerWidth;
+    if (w >= 1024) {
+      header.style.padding = '1.25% 0';
+      menu.style.display = 'none';
+      header.style.justifyContent = 'flex-end';
+      profile.style.width = '4%';
+    }
+  };
+  const profileMenu = (e) => {
+    let { target } = e;
+    while (target.className !== 'profile-icon') {
+      target = target.parentNode;
+    }
+    let signOut = document.querySelector('.sign-out-wrapper');
+    if (!signOut) {
+      const div = document.createElement('div');
+      div.className = 'sign-out-wrapper';
+      div.innerHTML = `<p class='sign-in-as'> Signed in as </p><p class='user-name'>${currentUser.username}</p><hr/><a href="/" class="sign-out">Sign out</a>`;
+      const wrapper = document.querySelector('.wrapper-app');
+      wrapper.appendChild(div);
+      wrapper.zIndex = 1;
+      div.zIndex = 999;
+    }
+    signOut = document.querySelector('.sign-out-wrapper');
+    signOut.style.display = 'flex';
+    const leaveBtn = document.querySelector('.sign-out');
+    leaveBtn.addEventListener('click', () => {
+      localStorage.clear();
+      dispatch({
+        type: 'LOGOUT',
+      });
+    });
+    document.addEventListener('click', (e) => {
+      if (e.target.closest('.profile-icon')) {
+        return;
+      }
+      if (e.target.closest('.sign-out-wrapper')) {
+        return;
+      }
+      signOut.style.display = 'none';
+    });
+  };
+  const handleDelete = (carid) => {
+    const url = `https://warm-inlet-48309.herokuapp.com/api/cars/${carid}`;
+    fetch(url, { method: 'DELETE' }).then((response) => {
+      if (response.status === 200) {
+        window.location.reload();
+      }
+    });
+    setLgShow(false);
+  };
+  const carData = useSelector((state) => state.Cars);
+  let cars = [];
+  if (Object.keys(carData).includes('cars')) {
+    cars = carData.cars;
+  }
+  useEffect(() => dispatch(fetchReserve()), []);
+  useEffect(() => dispatch(thunkUser()), []);
+  useEffect(() => dispatch(fetchAllCars()), []);
+  useEffect(() => adjustSize(), []);
+  useEffect(() => dispatch(thunkLikes()), []);
   return (
-    <h1>welcome react!</h1>
+    <div className="wrapper-app">
+      <Router>
+        <header
+          className="header"
+          style={{
+            display: session ? 'flex' : 'none',
+            backgroundColor: 'rgb(152, 191, 25)',
+            justifyContent: 'space-between',
+            alignItems: 'center',
+            padding: '5% 0',
+          }}
```

Comment:
- [ ] It's a good practice to do this using CSS instead of using the `style` attribute, so I suggest you to change it. If there are other parts of your code with this attribute, I recommend you to use CSS too.